src/Uno.UI/Generated/3.0.0.0/Windows.UI.Xaml.Controls/TextBlock.cs

```diff
@@ -371,13 +371,7 @@ public  void Select( global::Windows.UI.Xaml.Documents.TextPointer start,  globa
 			global::Windows.Foundation.Metadata.ApiInformation.TryRaiseNotImplemented("Windows.UI.Xaml.Controls.TextBlock", "void TextBlock.Select(TextPointer start, TextPointer end)");
 		}
 		#endif
-		#if __ANDROID__ || __IOS__ || NET461 || __WASM__ || __SKIA__ || __NETSTD_REFERENCE__ || __MACOS__
-		[global::Uno.NotImplemented("__ANDROID__", "__IOS__", "NET461", "__WASM__", "__SKIA__", "__NETSTD_REFERENCE__", "__MACOS__")]
-		public  bool Focus( global::Windows.UI.Xaml.FocusState value)
-		{
-			throw new global::System.NotImplementedException("The member bool TextBlock.Focus(FocusState value) is not implemented in Uno.");
-		}
-		#endif
+		// Skipping already declared method Windows.UI.Xaml.Controls.TextBlock.Focus(Windows.UI.Xaml.FocusState)
 		// Forced skipping of method Windows.UI.Xaml.Controls.TextBlock.SelectionHighlightColor.get
```

Comment:
@jeromelaban Here, the package diff tool is reporting removed API for Focus method. The method is technically still there (inherited from UIElement).

The root bug here is that UIElement in UWP doesn't have Focus method, while it does in Uno. What's the action to take here?

Reply:
@jeromelaban Can you help here please?

Reply:
We're going to be adjust this in the api alignment branch as well.

Reply:
@jeromelaban So how to move forward with this PR?

Reply:
We cannot merge this PR in the main branch, so we can only fix this on the API alignments one.

Reply:
@jeromelaban Is that really necessary? I mean, is the PackageDiff reported error is actually a breaking change?

The simplified case is like the following:

```diff
public class BaseClass
{
    public void M() { }
}

public class DerivedClass : BaseClass
{
-    public void M() { }
}
```

It's like the PackageDiff reporting a removed API here. However, applications will still compile. The runtime behavior may change, but it should be for the better behavior.

Reply:
Ping @jeromelaban

Reply:
@jeromelaban Can you take a quick look please?